cmake/onnxruntime_mlas.cmake

```diff
@@ -40,6 +40,7 @@ onnxruntime_add_static_library(onnxruntime_mlas
   ${MLAS_SRC_DIR}/reorder.cpp
   ${MLAS_SRC_DIR}/snchwc.cpp
   ${MLAS_SRC_DIR}/activate.cpp
+  ${MLAS_SRC_DIR}/activate_fp16.cpp
```

Comment:
Fixes linker error on Windows Arm64 build. Taken from https://github.com/microsoft/onnxruntime/pull/15213

Reply:
This is not the right fix, please see https://github.com/microsoft/onnxruntime/pull/15181 instead